docs/GDI-SOP_ISM.md

```diff
@@ -0,0 +1,176 @@
+2. OC/SCDP member shares template with the authors( OC/SDPC or nominated experts) to write the SOP (i.e. fill in the content)
+3. After the drafting is completed, the OC/SCDP member shares the draft internally or with experts to review for completeness.
+4. Both committees (OC/SDPC) approve the reviewed document.
+5. The OC/SDPC member then shares the approved SOP with GDI MB for authorization.
+6. GDI MB reviews the SOP and authorise or requests changes (repeat from Step 3)
+7. OC/SDPC accessions ([7.2](#72-gdi-sop-accessioning-and-file-namin)) the authorised SOP according to agreed process and the SOP goes into production
+8. OC/SDPC initiates periodic review cycle (repeat Steps 3-6)
+
+![Valid: Proposal for creating and approving European-level SOPs.](images/GDI-ISM_schema_European_Level.png)
+
+_Figure 1: Proposal for creating and approving European-level SOPs instances._
+
+#### 7.1.2 Node-specific SOP templates
+
+For each Node-specific SOP, first a template is created and approved at the European level, as explained above ([7.1.2](#712-node-specific-sop-templates)). Once approved, the template is then used by each node to create their own SOP instance ([7.1.3](#713-node-specific-sops)) which is approved at the node level. The steps below describe how to get a node-specific template created and approved.
+
+Steps:
+
+1. After a SOP request is received in the repository, a OC/SCDP member evaluates the request. If it is a valid request,the OC/SCDP member prepares the SOP draft from the [GDI General SOP template.](./GDI-SOP_sop-template.md)
+2. OC/SCDP member shares template with the authors( OC/SDPC or nominated experts) to write the SOP (i.e. fill in the content)
+3. After the drafting is completed, the OC/SCDP member shares the draft internally or with experts to review for completeness.
+4. Both committees (OC/SDPC) approve the reviewed document.
+5. The OC/SDPC member then shares the approved SOP with GDI MB for authorization.
+6. GDI MB reviews the SOP and authorise or requests changes (repeat from Step 3)
+7. OC/SDPC accessions ([7.2](#72-gdi-sop-accessioning-and-file-naming)) the approved SOP template according to agreed process and informs nodes ready to adapt to their own node processes.
+
+![Valid: Proposal for creating and approving node-specific SOP templates ](images/GDI-ISM_schema_Node_Template.png)
+
+_Figure 2: Proposal for creating and approving node-specific SOP templates._
+
+#### 7.1.3 Node-specific SOPs
+
+For each Node-specific SOP, first a template is created and approved at the European level ([7.1.2](#712-node-specific-sop-templates)). Once approved, the template is then used by each node to create their own SOP instance ([7.1.3](#713-node-specific-sops)) which is approved at the node level. The Steps below describe how nodes will create their node-specific SOPs.
+
+Steps:
+
+1. OC/SDPC representative(s) from each node copy the SOP template to their node’s SOP management system

```

Comment:
Here I would add a reference to the following document: https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_github-management.md

We should let it be known that their management system can very well be GitHub, so these can be cloned easily.